.github/workflows/dashboards-observability-test-and-build-workflow.yml

```diff
@@ -73,6 +73,115 @@ jobs:
       - name: Upload Artifact
         uses: actions/upload-artifact@v1
         with:
-          name: dashboards-observability
+          name: dashboards-observability-ubuntu
           path: ./OpenSearch-Dashboards/plugins/dashboards-observability/build
 
+  windows-build:
+
+    runs-on: windows-latest
+
+    steps:
+      - name: Checkout Plugin
+        uses: actions/checkout@v1
+
+      - name: Checkout OpenSearch Dashboards
+        uses: actions/checkout@v2
+        with:
+          repository: opensearch-project/OpenSearch-Dashboards
+          ref: ${{ env.OPENSEARCH_VERSION }}
+          path: OpenSearch-Dashboards
+
+      - name: Get node and yarn versions
+        id: versions_step
+        run: |
+          echo "::set-output name=node_version::$(node -p "(require('./OpenSearch-Dashboards/package.json').engines.node).match(/[.0-9]+/)[0]")"
+          echo "::set-output name=yarn_version::$(node -p "(require('./OpenSearch-Dashboards/package.json').engines.yarn).match(/[.0-9]+/)[0]")"
+
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ steps.versions_step.outputs.node_version }}
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Install correct yarn version for OpenSearch Dashboards
+        run: |
+          npm uninstall -g yarn
+          echo "Installing yarn ${{ steps.versions_step.outputs.yarn_version }}"
+          npm i -g yarn@${{ steps.versions_step.outputs.yarn_version }}
+
+      - name: Move Observability to Plugins Dir
+        run: mv dashboards-observability OpenSearch-Dashboards/plugins/dashboards-observability
+
+      - name: Plugin Bootstrap
+        run: |
+          cd OpenSearch-Dashboards/plugins/dashboards-observability
+          yarn osd bootstrap
+
```

Comment:
Skipped the test and upload coverage steps here. Had a random snapshot test failure on windows: 
```No result component › Renders No result component````
Not sure if it is ok to not run tests on windows and just assume it works since its the same thing.

Reply:
Test failure is here: 
https://github.com/opensearch-project/observability/actions/runs/3222406609/jobs/5271444877

Reply:
This is not transient. Can you check deeper in this? In my opinion we should not skip tests for any platform as that's the main purpose of CI pipelines. 

Skipping coverage is fine it's the same irrespective of the platform. We can just check the coverage for 1 platform.

Reply:
Yea It seems like windows is adding a newline somewhere, Im trying a few things to see if I can fix it....

Reply:
Fixed, and enabled unit tests for all platforms